config.default

```diff
@@ -132,6 +132,15 @@ mpasInterpolationMethod = bilinear
 startYear = 1
 endYear = 9999
 
+[index]
+## options related to producing nino index.
+
+# start and end years for the nino 3.4 analysis.  Using out-of-bounds values
+#   like start_year = 1 and end_year = 9999 will be clipped to the valid range
+#   of years, and is a good way of insuring that all values are used.
+startYear = 2
+endYear = 6
```

Comment:
Also, while I see the advantage of multiple config files for different machines.  I don't see the benefit of multiple files per machine.  Each file essentially only changes the run path, plot directory, and runName.  Doesn't seem like significant variability.